### ISSUE TITLE
Cannot connect after upgrading Ubuntu to 16.04

### ISSUE THREAD

**Issue by AdamFonagy** [FreeRDP/Remmina#946]
After upgrading to Ubuntu 16.04 I cannot connect to the RDP server earlier I could.

In the terminal I get this message when I connect with Remmina:
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.
[12:25:10:521] [16161:16204] [INFO][com.freerdp.client.common.cmdline] - loading channel cliprdr
[12:25:10:577] [16161:16204] [ERROR][com.freerdp.core] - freerdp_set_last_error ERRCONNECT_TLS_CONNECT_FAILED [0x20008]
[12:25:10:577] [16161:16204] [ERROR][com.freerdp.core.gateway.tsg] - rpc_connect error!
[12:25:10:577] [16161:16204] [ERROR][com.freerdp.core.nego] - Protocol Security Negotiation Failure
[12:25:10:577] [16161:16204] [ERROR][com.freerdp.core.connection] - Error: protocol security negotiation or connection failure
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.

And this when I try to connect with xFreeRDP:
[12:13:37:823] [16032:16033] [ERROR][com.freerdp.core] - freerdp_set_last_error ERRCONNECT_TLS_CONNECT_FAILED [0x20008]
[12:13:37:823] [16032:16033] [ERROR][com.freerdp.core.gateway.tsg] - rpc_connect error!
[12:13:37:824] [16032:16033] [ERROR][com.freerdp.core.nego] - Protocol Security Negotiation Failure
[12:13:37:824] [16032:16033] [ERROR][com.freerdp.core.connection] - Error: protocol security negotiation or connection failure
[12:13:37:824] [16032:16033] [ERROR][com.freerdp.client.x11] - Freerdp connect error exit status 1
- Client (OS name and version): Ubuntu x64 16.04
- Remmina version (remmina --version): Version 1.2.0-rcgit.14 (git n/a)
- Desktop environment (GNOME, Unity, KDE, ..): Unity
- Connecting to (OS and version): Windows Server
- Connecting via (RDP, VNC, ...): RDP
- RDP issues only, freerdp version (xfreerdp --version): version 2.0.0-dev (git n/a)
##

**Comment by AdamFonagy** [comment]
Meanwhile I tried many things and finally got to work an old version of FreeRDP  however remmina still doesn't work.
First I deleted the known_hosts file. It is located at ~/.config/freerdp on Ubuntu. After this I still got the same error with current GIT version of FreeRDP.
Then I downgraded FreeRDP to this version and got to work FreeRDP fine however remmina still doesn't work:

```
$ xfreerdp --version
This is FreeRDP version 1.2.5-dev (git 2ac195e)
```
The command line I used to get it work was this:
`xfreerdp /cert-ignore /f /v:hostname /u:myusername /d:mydomain /p:mypassword /g:addressofgateway /gu:myusername /gd:mydomain /gp:mypassword -nego /sec:rdp -encryption`

The host I connect to is a Windows Server 2008 R2.

I guess that Remmina doesn't pass the necessary parameters and switches to xfreerdp. And also there are params that I cannot set in Remmina like -nego, -encryption and auth params of the gateway.
